x-pack/plugins/infra/public/alerting/metric_threshold/components/expression_row.test.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mountWithIntl, nextTick } from 'test_utils/enzyme_helpers';
+import { MetricExpression } from '../types';
+import React from 'react';
+import { ExpressionRow } from './expression_row';
+import { act } from 'react-dom/test-utils';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { Comparator } from '../../../../server/lib/alerting/metric_threshold/types';
+
+jest.mock('../../../containers/source/use_source_via_http', () => ({
+  useSourceViaHttp: () => ({
+    source: { id: 'default' },
+    createDerivedIndexPattern: () => ({ fields: [], title: 'metricbeat-*' }),
+  }),
+}));
+
+describe('ExpressionRow', () => {
+  async function setup(expression: MetricExpression) {
+    const wrapper = mountWithIntl(
+      <ExpressionRow
+        canDelete={false}
+        fields={[
+          { name: 'system.cpu.user.pct', type: 'test' },
+          { name: 'system.load.1', type: 'test' },
+        ]}
+        remove={() => {}}
+        addExpression={() => {}}
+        key={1}
+        expressionId={1}
+        setAlertParams={() => {}}
+        errors={{
+          aggField: [],
+          timeSizeUnit: [],
+          timeWindowSize: [],
+        }}
+        expression={expression}
+      />
+    );
+
+    const update = async () =>
+      await act(async () => {
+        await nextTick();
+        wrapper.update();
+      });
+
+    await update();
+
+    return { wrapper, update };
+  }
+
+  it('should display thresholds as a percentage for pct metrics', async () => {
+    const expression = {
+      metric: 'system.cpu.user.pct',
+      comparator: Comparator.GT,
+      threshold: [0.5],
+      timeSize: 1,
+      timeUnit: 'm',
+      aggType: 'avg',
+    };
+    const { wrapper } = await setup(expression as MetricExpression);
+    const [valueMatch] = wrapper.html().match('<span class="euiExpression__value">50</span>') ?? [];
```

Comment:
@simianhacker This is the best I could get to actually run. Trying to match a `data-test-subj` value never brought up anything, the result of `wrapper.find()` always came out empty no matter what I put in there. Not sure what's up with our Enzyme install, it seems to be missing helpers that are present in the Enzyme docs

Reply:
How were you using `data-test-subj`? I was able to get it working by using `wrapper.find('[data-test-subj~="whatever"]')`

See: `x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/conditional_tooltip.test.tsx`

Reply:
I was looking for `alertThresholdInput` which is inside the `<ThresholdExpression>` component, but that never popped up. Nor did `thresholdPopover`, or any other `data-test-subj`s that I tried adding to any DOM wrappers